validator/checker.py

```diff
@@ -206,8 +207,10 @@ def check_single_version_update(
             if insert:
                 assert asset_data.cryptocompare not in seen_elements, f'Duplicate cryptocompare {asset_data.cryptocompare}'
                 assert asset_data.coingecko not in seen_elements, f'Duplicate coingecko {asset_data.coingecko}'
-                assert asset_data.name not in seen_elements, f'Duplicate name {asset_data.name}'
-                assert asset_data.symbol not in seen_elements, f'Duplicate symbol {asset_data.symbol}'
+                if asset_data.name in seen_elements:
```

Comment:
Why this change?

Reply:
Because there are two vaults with the same name and symbol

Reply:
This is not a good change. You are just adding a log warning which does nothing in tests. So this turns the check to a noop. What you should do is add an exception for these specific vaults only

Reply:
okey